common/reactive/pom.xml

```diff
@@ -81,5 +81,17 @@
             <artifactId>reactive-streams-tck-flow</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-core</artifactId>
+            <version>1.23</version>
```

Comment:
All dependencies must be managed. In this case this one should be managed under the top level pom.xml